features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1180,6 +1180,22 @@
             <Permissions>/permission/admin/manage/identity/applicationmgt/view</Permissions>
             <Scopes>internal_application_mgt_view</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/identity-providers(.*)" secured="true" http-method="POST">
+            <Permissions>/permission/admin/manage/identity/idpmgt/create</Permissions>
+            <Scopes>internal_idp_create</Scopes>
```

Comment:
Is this 'internal_idp_create' defined in the scope bindings xml?

Reply:
yes it is defined 
https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth-scope-bindings.xml#L364